### PR TITLE
Cosmetics (fix docstring triple quotes)

### PR DESCRIPTION
**Pull request by BigRoy**
## Changelog Description

Cosmetic fix where docstring accidentally started with four double quotes but closed with three.

## Additional info

Should not influence the running code.

## Testing notes:

1. No changes expected, so review the code.

### CODE REVIEW

File: server_addon/aftereffects/client/ayon_aftereffects/api/workfile_template_builder.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def _get_item(self, placeholder_item):
         return None, None
 
     def _collect_scene_placeholders(self):
-        """" Cache placeholder data to shared data.
+        """ Cache placeholder data to shared data.
```
Comment (iLLiCiTiT) [inline server_addon/aftereffects/client/ayon_aftereffects/api/workfile_template_builder.py:95]:
```suggestion
        """Cache placeholder data to shared data.
```

Comment (BigRoy) [reply server_addon/aftereffects/client/ayon_aftereffects/api/workfile_template_builder.py:95]: Thanks, wasn't sure whether that was some personal taste of somene. I saw a lot of this elsewhere - I think it was in the Nuke integration? Not sure how pissed you get by these spaces but if you need me I can make a separate cosmetic PR for those - hehe.

**Comment by BigRoy** [comment]
Side-note: do note that this is also tweaking vendorized files 🤦 - not sure if we want that?

File: server_addon/nuke/client/ayon_nuke/vendor/google/protobuf/text_format.py [hunk 2]
```diff
@@ -548,7 +548,7 @@ def PrintField(self, field, value):
     self.out.write(' ' if self.as_one_line else '\n')
 
   def _PrintShortRepeatedPrimitivesValue(self, field, value):
-    """"Prints short repeated primitives value."""
+    """Prints short repeated primitives value."""
```
Comment (iLLiCiTiT) [inline server_addon/nuke/client/ayon_nuke/vendor/google/protobuf/text_format.py:551]:
```suggestion
    """"Prints short repeated primitives value."""
```

File: server_addon/hiero/client/ayon_hiero/vendor/google/protobuf/text_format.py [hunk 3]
```diff
@@ -548,7 +548,7 @@ def PrintField(self, field, value):
     self.out.write(' ' if self.as_one_line else '\n')
 
   def _PrintShortRepeatedPrimitivesValue(self, field, value):
-    """"Prints short repeated primitives value."""
+    """Prints short repeated primitives value."""
```
Comment (iLLiCiTiT) [inline server_addon/hiero/client/ayon_hiero/vendor/google/protobuf/text_format.py:551]:
```suggestion
    """"Prints short repeated primitives value."""
```

**Comment by iLLiCiTiT** [comment]
> Side-note: do note that this is also tweaking vendorized files 🤦 - not sure if we want that?

Didn't realize, nope. Should not happen :)

**Comment by BigRoy** [comment]
> Didn't realize, nope. Should not happen :)

Didn't notice it myself at first either - anyway, thanks. Fixed now.

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**